packages/expo-cli/src/commands/eas-build/build/utils/misc.ts

```diff
@@ -4,6 +4,11 @@ import log from '../../../../log';
 import * as UrlUtils from '../../../utils/url';
 import { Build } from '../../types';
 
+const platformMapName = {
+  ios: 'iOS',
+  android: 'Android',
+};
```

Comment:
It's not a util function. Create `constants.ts` (or similar) and move it there.

Reply:
It is util, it just not a function, but ok